server/src/main/java/com/google/musicanalysis/site/YoutubeGenres.java

```diff
@@ -0,0 +1,104 @@
+package com.google.musicanalysis.site;
+
+import com.google.gson.JsonParser;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonArray;
+import com.google.gson.Gson;
+import java.util.HashMap;
+import java.util.Map;
+import java.lang.Math;
+
+enum MusicTopic {
+  MUSIC_ONLY,
+  MUSIC_SUBGENRE,
+  NOT_MUSIC
+}
+
+/** contains final object that YoutubeServlet.java sends to frontend */
+public class YoutubeGenres {
+  private HashMap<String, Integer> genreData = new HashMap<String, Integer>();
+  protected int totalLiked;
+  private int totalMusic = 0;
+  private boolean isMusic;
+  private int maxGenreCount = 0;
```

Comment:
`totalMusic` and `maxGenreCount` seem unused. Can we delete them?

Reply:
I update `totalMusic` outside this file (in YoutubeServlet.java) and 

`maxGenreCount` within the `updateGenre` function within YoutubeGenres.java

Reply:
I need to use `maxGenreCount` to determine the scaling of the bar chart in HTML frontend.
I also need to use totalMusic to figure out the size of the heat map in the frontend.

Reply:
I need to use both variables in the frontend.
totalMusic will be displayed in frontend as a stat.
maxGenreCount determines bar chart lengths.

Reply:
I see. It's because you're turning this class into json. Except, that means you're probably sending more values to the front end than you want, and in practice these values /are/ public.

If Jess and I didn't realize that this was a necessary variable, then other developers in the future would likely have the same intuition, remove the values, and end up breaking the app. I suggest making these variables public, and composing a new json object to return made up of just your public members.

Reply:
Discussed offline. The resolution is to make all the class members public since they're all used in the front end and are meant to be serialized.